src/common/templates/_dbv3.tpl

```diff
@@ -184,6 +184,7 @@ OPTIONAL:
                 {{- $hosts := $localDBCtx.hosts }}
                 {{- $protocol := $localDBCtx.protocol }}
                 {{- $extraArgs := $localDBCtx.extraArgs }}
+                {{- $database = default $database $localDBCtx.database }}
```

Comment:
this is done because: mongo-uri in saas may not have pms-harness format, it might be harness-pms, pms-harness-prod